diff --git viewer/BackRenderingWindow.java viewer/BackRenderingWindow.java
index 81f7837..ab0037e 100644
--- viewer/BackRenderingWindow.java
+++ viewer/BackRenderingWindow.java
@@ -9,9 +9,9 @@ import javax.swing.event.*;
 
 import javax.media.opengl.*;
 import javax.media.opengl.glu.*;
-import com.jogamp.opengl.util.*;
+import com.sun.opengl.util.*;
 
-import com.jogamp.opengl.util.gl2.*;
+import com.sun.opengl.util.gl2.*;
 import javax.media.opengl.awt.*;
 
 
diff --git viewer/RenderingWindow.java viewer/RenderingWindow.java
index 60397d2..d193788 100644
--- viewer/RenderingWindow.java
+++ viewer/RenderingWindow.java
@@ -9,9 +9,9 @@ import javax.swing.event.*;
 
 import javax.media.opengl.*;
 import javax.media.opengl.glu.*;
-import com.jogamp.opengl.util.*;
+import com.sun.opengl.util.*;
 
-import com.jogamp.opengl.util.gl2.*;
+import com.sun.opengl.util.gl2.*;
 import javax.media.opengl.awt.*;
 
 
@@ -1376,7 +1376,6 @@ public class RenderingWindow extends JFrame implements GLEventListener,
     resetFPS();
   }
   public void resetFPS(){
-    animator.remove(drawable);
     //if(!isConstuctorFinished)return;
     if( isAnimating() ){
       animator.stop();
diff --git viewer/renderer/Annotation.java viewer/renderer/Annotation.java
index d0ccb61..e6f123f 100644
--- viewer/renderer/Annotation.java
+++ viewer/renderer/Annotation.java
@@ -10,10 +10,10 @@ import javax.swing.event.*;
 
 import javax.media.opengl.*;
 import javax.media.opengl.glu.*;
-import com.jogamp.opengl.util.*;
-import com.jogamp.opengl.util.awt.*;
+import com.sun.opengl.util.*;
+import com.sun.opengl.util.awt.*;
 
-import com.jogamp.opengl.util.gl2.*;
+import com.sun.opengl.util.gl2.*;
 import javax.media.opengl.awt.*;
 
 import viewer.*;
diff --git viewer/renderer/AtomLabel.java viewer/renderer/AtomLabel.java
index 7704e7c..1c8943a 100644
--- viewer/renderer/AtomLabel.java
+++ viewer/renderer/AtomLabel.java
@@ -9,9 +9,9 @@ import javax.swing.event.*;
 
 import javax.media.opengl.*;
 import javax.media.opengl.glu.*;
-import com.jogamp.opengl.util.*;
+import com.sun.opengl.util.*;
 
-import com.jogamp.opengl.util.gl2.*;
+import com.sun.opengl.util.gl2.*;
 import javax.media.opengl.awt.*;
 
 import tools.*;
diff --git viewer/renderer/AtomSelector.java viewer/renderer/AtomSelector.java
index c84e0b0..d65359c 100644
--- viewer/renderer/AtomSelector.java
+++ viewer/renderer/AtomSelector.java
@@ -5,9 +5,9 @@ import java.nio.*;
 
 import javax.media.opengl.*;
 import javax.media.opengl.glu.*;
-import com.jogamp.opengl.util.*;
+import com.sun.opengl.util.*;
 
-import com.jogamp.opengl.util.gl2.*;
+import com.sun.opengl.util.gl2.*;
 import javax.media.opengl.awt.*;
 
 import viewer.*;
diff --git viewer/renderer/Atoms.java viewer/renderer/Atoms.java
index 0e3592b..95b7ef6 100644
--- viewer/renderer/Atoms.java
+++ viewer/renderer/Atoms.java
@@ -9,9 +9,9 @@ import javax.swing.event.*;
 
 import javax.media.opengl.*;
 import javax.media.opengl.glu.*;
-import com.jogamp.opengl.util.*;
+import com.sun.opengl.util.*;
 
-import com.jogamp.opengl.util.gl2.*;
+import com.sun.opengl.util.gl2.*;
 import javax.media.opengl.awt.*;
 
 import tools.*;
diff --git viewer/renderer/Axis.java viewer/renderer/Axis.java
index 7ab7f8f..9ce8ddf 100644
--- viewer/renderer/Axis.java
+++ viewer/renderer/Axis.java
@@ -8,10 +8,10 @@ import javax.swing.event.*;
 
 import javax.media.opengl.*;
 import javax.media.opengl.glu.*;
-import com.jogamp.opengl.util.*;
-import com.jogamp.opengl.util.awt.*;
+import com.sun.opengl.util.*;
+import com.sun.opengl.util.awt.*;
 
-import com.jogamp.opengl.util.gl2.*;
+import com.sun.opengl.util.gl2.*;
 import javax.media.opengl.awt.*;
 
 import viewer.*;
diff --git viewer/renderer/Bonds.java viewer/renderer/Bonds.java
index 5e0c049..7fdd332 100644
--- viewer/renderer/Bonds.java
+++ viewer/renderer/Bonds.java
@@ -11,9 +11,9 @@ import javax.swing.table.*;
 
 import javax.media.opengl.*;
 import javax.media.opengl.glu.*;
-import com.jogamp.opengl.util.*;
+import com.sun.opengl.util.*;
 
-import com.jogamp.opengl.util.gl2.*;
+import com.sun.opengl.util.gl2.*;
 import javax.media.opengl.awt.*;
 
 import viewer.*;
diff --git viewer/renderer/Box.java viewer/renderer/Box.java
index 5055b22..c4365c0 100644
--- viewer/renderer/Box.java
+++ viewer/renderer/Box.java
@@ -8,9 +8,9 @@ import javax.swing.event.*;
 
 import javax.media.opengl.*;
 import javax.media.opengl.glu.*;
-import com.jogamp.opengl.util.*;
+import com.sun.opengl.util.*;
 
-import com.jogamp.opengl.util.gl2.*;
+import com.sun.opengl.util.gl2.*;
 import javax.media.opengl.awt.*;
 
 import viewer.*;
diff --git viewer/renderer/Light.java viewer/renderer/Light.java
index 724dd9a..25862da 100644
--- viewer/renderer/Light.java
+++ viewer/renderer/Light.java
@@ -8,9 +8,9 @@ import javax.swing.border.*;
 
 import javax.media.opengl.*;
 import javax.media.opengl.glu.*;
-import com.jogamp.opengl.util.*;
+import com.sun.opengl.util.*;
 
-import com.jogamp.opengl.util.gl2.*;
+import com.sun.opengl.util.gl2.*;
 import javax.media.opengl.awt.*;
 
 import viewer.*;
diff --git viewer/renderer/Plane.java viewer/renderer/Plane.java
index 0b84e8f..5ca8d80 100644
--- viewer/renderer/Plane.java
+++ viewer/renderer/Plane.java
@@ -9,9 +9,9 @@ import java.io.*;
 
 import javax.media.opengl.*;
 import javax.media.opengl.glu.*;
-import com.jogamp.opengl.util.*;
+import com.sun.opengl.util.*;
 
-import com.jogamp.opengl.util.gl2.*;
+import com.sun.opengl.util.gl2.*;
 import javax.media.opengl.awt.*;
 
 
diff --git viewer/renderer/Ring.java viewer/renderer/Ring.java
index f931f28..c701a91 100644
--- viewer/renderer/Ring.java
+++ viewer/renderer/Ring.java
@@ -8,13 +8,13 @@ import javax.swing.event.*;
 
 import javax.media.opengl.*;
 import javax.media.opengl.glu.*;
-import com.jogamp.opengl.util.*;
+import com.sun.opengl.util.*;
 
-import com.jogamp.opengl.util.gl2.*;
+import com.sun.opengl.util.gl2.*;
 import javax.media.opengl.awt.*;
 
 
-import com.jogamp.opengl.util.texture.*;
+import com.sun.opengl.util.texture.*;
 import java.io.*;
 import java.util.*;
 
diff --git viewer/renderer/RotationCenter.java viewer/renderer/RotationCenter.java
index 6c5c4cf..564dc80 100644
--- viewer/renderer/RotationCenter.java
+++ viewer/renderer/RotationCenter.java
@@ -8,13 +8,13 @@ import javax.swing.event.*;
 
 import javax.media.opengl.*;
 import javax.media.opengl.glu.*;
-import com.jogamp.opengl.util.*;
+import com.sun.opengl.util.*;
 
-import com.jogamp.opengl.util.gl2.*;
+import com.sun.opengl.util.gl2.*;
 import javax.media.opengl.awt.*;
 
 
-import com.jogamp.opengl.util.texture.*;
+import com.sun.opengl.util.texture.*;
 import java.io.*;
 import java.util.*;
 
diff --git viewer/renderer/SelectorQueue.java viewer/renderer/SelectorQueue.java
index 62f5eaa..eff0457 100644
--- viewer/renderer/SelectorQueue.java
+++ viewer/renderer/SelectorQueue.java
@@ -2,9 +2,9 @@ package viewer.renderer;
 import java.util.*;
 import javax.media.opengl.*;
 import javax.media.opengl.glu.*;
-import com.jogamp.opengl.util.*;
+import com.sun.opengl.util.*;
 
-import com.jogamp.opengl.util.gl2.*;
+import com.sun.opengl.util.gl2.*;
 import javax.media.opengl.awt.*;
 
 import viewer.*;
diff --git viewer/renderer/Snapshot.java viewer/renderer/Snapshot.java
index c93549b..b87743a 100644
--- viewer/renderer/Snapshot.java
+++ viewer/renderer/Snapshot.java
@@ -13,12 +13,12 @@ import javax.imageio.*;
 
 import javax.media.opengl.*;
 import javax.media.opengl.glu.*;
-import com.jogamp.opengl.util.*;
+import com.sun.opengl.util.*;
 
-import com.jogamp.opengl.util.gl2.*;
+import com.sun.opengl.util.gl2.*;
 import javax.media.opengl.awt.*;
 
-import com.jogamp.opengl.util.awt.*;
+import com.sun.opengl.util.awt.*;
 
 
 import viewer.*;
diff --git viewer/renderer/Toon.java viewer/renderer/Toon.java
index f921675..e422ca6 100644
--- viewer/renderer/Toon.java
+++ viewer/renderer/Toon.java
@@ -6,10 +6,10 @@ import java.util.*;
 import javax.media.opengl.*;
 import javax.media.opengl.glu.*;
 import javax.media.opengl.awt.*;
-import com.jogamp.opengl.util.*;
-import com.jogamp.opengl.util.gl2.*;
-import com.jogamp.opengl.util.awt.*;
-import com.jogamp.opengl.util.glsl.*;
+import com.sun.opengl.util.*;
+import com.sun.opengl.util.gl2.*;
+import com.sun.opengl.util.awt.*;
+import com.sun.opengl.util.glsl.*;
 
 public class Toon {
 
@@ -72,7 +72,7 @@ public class Toon {
     ShaderCode rsFp = new ShaderCode( GL2ES2.GL_FRAGMENT_SHADER,
                                       1,
                                       readShaderCode(fpfile) );
-    /*
+    /* 
      * ShaderCode rsVp = ShaderCode.create( glsl, GL2ES2.GL_VERTEX_SHADER,
      *                                      1, RenderingWindow.class,
      *                                      "viewer/renderer/src", "bin",
@@ -93,10 +93,10 @@ public class Toon {
   }
 
   public void set() {
-    st.useProgram( glsl, true );
+    st.glUseProgram( glsl, true );
   }
   public void unset() {
-    st.useProgram( glsl, false );
+    st.glUseProgram( glsl, false );
   }
 
 }
diff --git viewer/renderer/Vectors.java viewer/renderer/Vectors.java
index 4f107bf..8057b21 100644
--- viewer/renderer/Vectors.java
+++ viewer/renderer/Vectors.java
@@ -11,9 +11,9 @@ import javax.swing.table.*;
 
 import javax.media.opengl.*;
 import javax.media.opengl.glu.*;
-import com.jogamp.opengl.util.*;
+import com.sun.opengl.util.*;
 
-import com.jogamp.opengl.util.gl2.*;
+import com.sun.opengl.util.gl2.*;
 import javax.media.opengl.awt.*;
 
 import data.*;
diff --git viewer/renderer/Viewpoint.java viewer/renderer/Viewpoint.java
index f0348fa..f72b510 100644
--- viewer/renderer/Viewpoint.java
+++ viewer/renderer/Viewpoint.java
@@ -9,9 +9,9 @@ import java.io.*;
 
 import javax.media.opengl.*;
 import javax.media.opengl.glu.*;
-import com.jogamp.opengl.util.*;
+import com.sun.opengl.util.*;
 
-import com.jogamp.opengl.util.gl2.*;
+import com.sun.opengl.util.gl2.*;
 import javax.media.opengl.awt.*;
 
 import viewer.*;
diff --git viewer/renderer/Volume.java viewer/renderer/Volume.java
index 3307728..f53231d 100644
--- viewer/renderer/Volume.java
+++ viewer/renderer/Volume.java
@@ -11,10 +11,10 @@ import javax.swing.table.*;
 
 import javax.media.opengl.*;
 import javax.media.opengl.glu.*;
-import com.jogamp.opengl.util.*;
+import com.sun.opengl.util.*;
 
 
-import com.jogamp.opengl.util.gl2.*;
+import com.sun.opengl.util.gl2.*;
 import javax.media.opengl.awt.*;
 
 import viewer.*;
diff --git viewer/renderer/atlantis/Atlantis.java viewer/renderer/atlantis/Atlantis.java
index 98790ae..d42c86f 100644
--- viewer/renderer/atlantis/Atlantis.java
+++ viewer/renderer/atlantis/Atlantis.java
@@ -2,9 +2,9 @@ package viewer.renderer.atlantis;
 
 import javax.media.opengl.*;
 import javax.media.opengl.glu.*;
-import com.jogamp.opengl.util.*;
+import com.sun.opengl.util.*;
 
-import com.jogamp.opengl.util.gl2.*;
+import com.sun.opengl.util.gl2.*;
 import javax.media.opengl.awt.*;
 
 import viewer.renderer.atlantis.*;
diff --git viewer/renderer/atlantis/Dolphin.java viewer/renderer/atlantis/Dolphin.java
index 28ad424..a57f4e4 100644
--- viewer/renderer/atlantis/Dolphin.java
+++ viewer/renderer/atlantis/Dolphin.java
@@ -2,9 +2,9 @@ package viewer.renderer.atlantis;
 
 import javax.media.opengl.*;
 import javax.media.opengl.glu.*;
-import com.jogamp.opengl.util.*;
+import com.sun.opengl.util.*;
 
-import com.jogamp.opengl.util.gl2.*;
+import com.sun.opengl.util.gl2.*;
 import javax.media.opengl.awt.*;
 
 /**
diff --git viewer/renderer/atlantis/Shark.java viewer/renderer/atlantis/Shark.java
index 4475476..2c1a6f7 100644
--- viewer/renderer/atlantis/Shark.java
+++ viewer/renderer/atlantis/Shark.java
@@ -2,9 +2,9 @@ package viewer.renderer.atlantis;
 
 import javax.media.opengl.*;
 import javax.media.opengl.glu.*;
-import com.jogamp.opengl.util.*;
+import com.sun.opengl.util.*;
 
-import com.jogamp.opengl.util.gl2.*;
+import com.sun.opengl.util.gl2.*;
 import javax.media.opengl.awt.*;
 
 /**
diff --git viewer/renderer/atlantis/Whale.java viewer/renderer/atlantis/Whale.java
index 0ad54be..e208459 100644
--- viewer/renderer/atlantis/Whale.java
+++ viewer/renderer/atlantis/Whale.java
@@ -2,9 +2,9 @@ package viewer.renderer.atlantis;
 
 import javax.media.opengl.*;
 import javax.media.opengl.glu.*;
-import com.jogamp.opengl.util.*;
+import com.sun.opengl.util.*;
 
-import com.jogamp.opengl.util.gl2.*;
+import com.sun.opengl.util.gl2.*;
 import javax.media.opengl.awt.*;
 
 
diff --git viewer/viewConfigPanel/ComboPanel.java viewer/viewConfigPanel/ComboPanel.java
index 276dac7..0e75e51 100644
--- viewer/viewConfigPanel/ComboPanel.java
+++ viewer/viewConfigPanel/ComboPanel.java
@@ -10,9 +10,9 @@ import javax.swing.table.*;
 
 import javax.media.opengl.*;
 import javax.media.opengl.glu.*;
-import com.jogamp.opengl.util.*;
+import com.sun.opengl.util.*;
 
-import com.jogamp.opengl.util.gl2.*;
+import com.sun.opengl.util.gl2.*;
 import javax.media.opengl.awt.*;
 
 import tools.*;
diff --git viewer/viewConfigPanel/ExportPanel.java viewer/viewConfigPanel/ExportPanel.java
index dc4c87d..8f089e0 100644
--- viewer/viewConfigPanel/ExportPanel.java
+++ viewer/viewConfigPanel/ExportPanel.java
@@ -12,9 +12,9 @@ import java.lang.reflect.Method;
 
 import javax.media.opengl.*;
 import javax.media.opengl.glu.*;
-import com.jogamp.opengl.util.*;
+import com.sun.opengl.util.*;
 
-import com.jogamp.opengl.util.gl2.*;
+import com.sun.opengl.util.gl2.*;
 import javax.media.opengl.awt.*;
 
 import data.*;
diff --git viewer/viewConfigPanel/MDPanel.java viewer/viewConfigPanel/MDPanel.java
index a25b1c5..514b285 100644
--- viewer/viewConfigPanel/MDPanel.java
+++ viewer/viewConfigPanel/MDPanel.java
@@ -11,10 +11,10 @@ import java.lang.reflect.Method;
 
 import javax.media.opengl.*;
 import javax.media.opengl.glu.*;
-import com.jogamp.opengl.util.*;
-import com.jogamp.opengl.util.awt.*;
+import com.sun.opengl.util.*;
+import com.sun.opengl.util.awt.*;
 
-import com.jogamp.opengl.util.gl2.*;
+import com.sun.opengl.util.gl2.*;
 import javax.media.opengl.awt.*;
 
 import viewer.viewConfigPanel.plugin.*;
diff --git viewer/viewConfigPanel/ModelingPanel.java viewer/viewConfigPanel/ModelingPanel.java
index c8f1415..643c5cc 100644
--- viewer/viewConfigPanel/ModelingPanel.java
+++ viewer/viewConfigPanel/ModelingPanel.java
@@ -11,10 +11,10 @@ import java.lang.reflect.Method;
 
 import javax.media.opengl.*;
 import javax.media.opengl.glu.*;
-import com.jogamp.opengl.util.*;
-import com.jogamp.opengl.util.awt.*;
+import com.sun.opengl.util.*;
+import com.sun.opengl.util.awt.*;
 
-import com.jogamp.opengl.util.gl2.*;
+import com.sun.opengl.util.gl2.*;
 import javax.media.opengl.awt.*;
 
 import viewer.viewConfigPanel.plugin.*;
diff --git viewer/viewConfigPanel/NeighborAnalysisPanel.java viewer/viewConfigPanel/NeighborAnalysisPanel.java
index 65d2c50..e147c4f 100644
--- viewer/viewConfigPanel/NeighborAnalysisPanel.java
+++ viewer/viewConfigPanel/NeighborAnalysisPanel.java
@@ -9,9 +9,9 @@ import javax.swing.event.*;
 
 import javax.media.opengl.*;
 import javax.media.opengl.glu.*;
-import com.jogamp.opengl.util.*;
+import com.sun.opengl.util.*;
 
-import com.jogamp.opengl.util.gl2.*;
+import com.sun.opengl.util.gl2.*;
 import javax.media.opengl.awt.*;
 
 import data.*;
diff --git viewer/viewConfigPanel/PlotterPanel.java viewer/viewConfigPanel/PlotterPanel.java
index 8d7ec2b..8f5829a 100644
--- viewer/viewConfigPanel/PlotterPanel.java
+++ viewer/viewConfigPanel/PlotterPanel.java
@@ -9,9 +9,9 @@ import javax.swing.event.*;
 
 import javax.media.opengl.*;
 import javax.media.opengl.glu.*;
-import com.jogamp.opengl.util.*;
+import com.sun.opengl.util.*;
 
-import com.jogamp.opengl.util.gl2.*;
+import com.sun.opengl.util.gl2.*;
 import javax.media.opengl.awt.*;
 
 import data.*;
diff --git viewer/viewConfigPanel/RadialDistributionPanel.java viewer/viewConfigPanel/RadialDistributionPanel.java
index 93448be..ee46de0 100644
--- viewer/viewConfigPanel/RadialDistributionPanel.java
+++ viewer/viewConfigPanel/RadialDistributionPanel.java
@@ -9,9 +9,9 @@ import javax.swing.event.*;
 
 import javax.media.opengl.*;
 import javax.media.opengl.glu.*;
-import com.jogamp.opengl.util.*;
+import com.sun.opengl.util.*;
 
-import com.jogamp.opengl.util.gl2.*;
+import com.sun.opengl.util.gl2.*;
 import javax.media.opengl.awt.*;
 
 import data.*;
diff --git viewer/viewConfigPanel/TrajectoryPanel.java viewer/viewConfigPanel/TrajectoryPanel.java
index 45cebf6..72f1ae7 100644
--- viewer/viewConfigPanel/TrajectoryPanel.java
+++ viewer/viewConfigPanel/TrajectoryPanel.java
@@ -11,9 +11,9 @@ import javax.swing.table.*;
 
 import javax.media.opengl.*;
 import javax.media.opengl.glu.*;
-import com.jogamp.opengl.util.*;
+import com.sun.opengl.util.*;
 
-import com.jogamp.opengl.util.gl2.*;
+import com.sun.opengl.util.gl2.*;
 import javax.media.opengl.awt.*;
 
 import viewer.*;
diff --git viewer/viewConfigPanel/ViewPointPanel.java viewer/viewConfigPanel/ViewPointPanel.java
index 5ba98a3..584b7e9 100644
--- viewer/viewConfigPanel/ViewPointPanel.java
+++ viewer/viewConfigPanel/ViewPointPanel.java
@@ -10,9 +10,9 @@ import javax.swing.event.*;
 
 import javax.media.opengl.*;
 import javax.media.opengl.glu.*;
-import com.jogamp.opengl.util.*;
+import com.sun.opengl.util.*;
 
-import com.jogamp.opengl.util.gl2.*;
+import com.sun.opengl.util.gl2.*;
 import javax.media.opengl.awt.*;
 
 import data.*;
